android/Samples/KMSample2/build.sh

```diff
@@ -1,10 +1,61 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

This gets the newer version of bash on macOS when installed with homebrew and works correctly on other platforms also.